assets/quick-buy.css

```diff
@@ -0,0 +1,197 @@
+  padding: 0 1.5rem 0 3rem;
+  border-color: rgba(var(--color-foreground), var(--popup-border-opacity));
+  border-style: solid;
+  border-width: var(--popup-border-width);
+  box-shadow: var(--popup-shadow-horizontal-offset) var(--popup-shadow-vertical-offset) var(--popup-shadow-blur-radius) rgba(var(--color-shadow), var(--popup-shadow-opacity));
+}
+
+.quick-buy-modal__content.focused {
+  box-shadow: 0 0 0 0.3rem rgb(var(--color-background)),
+  0 0 0.5rem 0.4rem rgba(var(--color-foreground), 0.3),
+  var(--popup-shadow-horizontal-offset) var(--popup-shadow-vertical-offset) var(--popup-shadow-blur-radius) rgba(var(--color-shadow), var(--popup-shadow-opacity));
+}
+
+.quick-buy-modal__content:focus-visible{
+  box-shadow: 0 0 0 0.3rem rgb(var(--color-background)),
+  0 0 0.5rem 0.4rem rgba(var(--color-foreground), 0.3),
+  var(--popup-shadow-horizontal-offset) var(--popup-shadow-vertical-offset) var(--popup-shadow-blur-radius) rgba(var(--color-shadow), var(--popup-shadow-opacity));
+}
+
+@media screen and (min-width: 750px) {
+  .quick-buy-modal__content {
+    padding-right: 1.5rem;
+    margin-top: 10rem;
+    width: 70%;
+    padding: 0 3rem 2rem 3rem;
+  }
+}
+
+.quick-buy-modal__content img {
+  max-width: 100%;
+}
+
+.quick-buy-modal__content-info {
+  padding-right: 4.4rem;
+}
+
+.quick-buy-modal__content-info > * {
+  height: auto;
+  margin: 0 auto;
+  max-width: 100%;
+  width: 100%;
+}
+
+@media screen and (max-width: 749px) {
+  .quick-buy-modal__content-info > * {
+    max-height: 100%;
+  }
+}
+
+.quick-buy-modal__toggle {
+  background-color: rgb(var(--color-background));
+  border: 0.1rem solid rgba(var(--color-foreground), 0.1);
+  border-radius: 50%;
+  color: rgba(var(--color-foreground), 0.55);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+  position: sticky;
+  padding: 1.2rem;
+  z-index: 5;
+  top: 1.5rem;
+  width: 4rem;
+  margin: 0 0 0 auto;
+  margin-bottom: -0.8rem;
+}
+
+.quick-buy-modal__toggle:hover {
+  color: rgba(var(--color-foreground), 0.75);
+}
+
+.quick-buy-modal__toggle .icon {
+  height: auto;
+  margin: 0;
+  width: 2.2rem;
+}
+
+.quick-buy-modal__content {
+  max-width: var(--page-width);
+}
+
+quickbuy-modal .quick-buy-modal__content-info {
+  padding-right: 0;
+}
+
+quickbuy-modal .product:not(.featured-product) .product__view-details {
+  display: block;
+}
+
+quickbuy-modal .product__media-toggle,
+quickbuy-modal .skip-to-content-link,
+quickbuy-modal .product__media-icon,
+quickbuy-modal .slider-buttons,
+quickbuy-modal .thumbnail-list,
+quickbuy-modal .product-popup-modal__opener,
+quickbuy-modal .product__modal-opener:not(.product__modal-opener--image),
+quickbuy-modal .product__media-item:not(:first-child),
+quickbuy-modal pickup-availability,
+quickbuy-modal .product__description,
+quickbuy-modal .product__accordion,
+quickbuy-modal .share-button,
+quickbuy-modal .slider-button,
+quickbuy-modal product-modal {
+  display: none !important;
+}
+
```

Comment:
There is something to figure out I think on mobile for the `view in your space` button. In this example there is a 3d model but it's not associated to the variants. So it creates extra space and shouldn't be there since from the modal you won't be able to get to the 3d model. 
![](https://screenshot.click/03-26-pu78r-x8gnm.png)

Reply:
are you looking at this from the editor or store? Because I remember catching something like this before, and I think @LucasLacerdaUX told me that this doesn't happen on store.

Reply:
Yep! I double checked on my phone to confirm 👍

Reply:
Good catch. I'll check why it's not working 👀